NEWS.rst

```diff
@@ -21,6 +57,9 @@ New Features
 
 - --is-visible option returns 1 when visible, and 0 when not
 
+- - Double click to open a new tab, without side effects in mouse
```

Comment:
Is the duplicate `-` cause by original git commit?

Reply:
It is just the slug file to edit (this can be done in a new commit)

Reply:
The file is unmodified `make release-note` output. The dash comes from the reno file also having a dash.

Reply:
File is restore_double_click_open_tab-c4dfc824ce4b5e85.yaml